Assets/MixedRealityToolkit/_Core/DataProviders/SpatialObservers/WindowsMixedReality/WindowsMixedRealitySpatialMeshObserver.cs

```diff
@@ -141,9 +138,9 @@ public override void Destroy()
         /// <summary> 
         /// To prevent too many meshes from being generated at the same time, we will 
         /// only request one mesh to be created at a time.  This variable will track 
-        /// if a mesh creation request is in flight. 
+        /// if a mesh creation request is in flight.
         /// </summary> 
-        private readonly Dictionary<int, SpatialMeshObject> outstandingMeshObjects = new Dictionary<int, SpatialMeshObject>();
+        private SpatialMeshObject? pendingSpatialObject = null;
```

Comment:
Can you please also restore the spareSpatialObject field and associated logic? It helps preserve performance via object reuse vs recreation.

```
        /// <summary>
        /// When surfaces are replaced or removed, rather than destroying them, we'll keep
        /// one as a spare for use in outstanding mesh requests. That way, we'll have fewer
        /// game object create/destroy cycles, which should help performance.
        /// </summary>
        protected SpatialMeshObject? spareMeshObject = null;```

Reply:
I don't understand. Can you open a pr with the suggested edits?

Reply:
Will do.